### PR TITLE
5-get-api-articles finished

### PR DESCRIPTION
**Pull request by DexterGundar**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const db = require("../db/connection.js");
 const seed = require("../db/seeds/seed.js");
 const testData = require("../db/data/test-data");
 const endPoints = require("../endpoints.json");
+const jestSorted = require('jest-sorted')
```
Comment (RichardNeat) [inline __tests__/app.test.js:8]: According to the jest-sorted docs, there is no need to require anything in here. We aren't using this jestSorted variable which is our clue.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -84,4 +83,44 @@ describe('get api',()=>{
...
+              expect(article).toHaveProperty("topic", expect.any(String)),
+              expect(article).toHaveProperty("created_at", expect.any(String)),
+              expect(article).toHaveProperty("votes", expect.any(Number)),
+              expect(article).toHaveProperty("article_img_url", expect.any(String)),
+              expect(article).toHaveProperty("comment_count", expect.any(String)),
+              expect(article).not.toHaveProperty('body')
+            })
+      })
+  });
+  test("return 404, if non-existant address has been entered", () => {
```
Comment (RichardNeat) [inline __tests__/app.test.js:108]: In our server, we only need one bad path test. This should not be inside this describe block as it has nothing to do with this endpoint and doesn't access the controller or model for this endpoint. There is no functional different between /api/article and /api/banana :)
Please delete if you already have a bad path test, or if you're adding it for the first time now just pop it in its own describe block.

**Review by RichardNeat — CHANGES_REQUESTED** [review]
Nice one! Just a couple of suggested changes in comments below to look at please then you're good to go.

File: __tests__/app.test.js (hunk 1 above)
Comment (DexterGundar) [reply __tests__/app.test.js:8]: when I remove the jest sorted testing gives me: TypeError: expect(...).toBeSortedBy is not a function

**Comment by DexterGundar** [comment]
added to package.json jest-sorted - test works now.

**Review by Jodiech — APPROVED** [review]
Nice work! Happy for you to merge 👍

**Pull request merged**